waku/waku_lightpush/self_req_handler.nim

```diff
@@ -0,0 +1,54 @@
+##
+## This file is aimed to attend the requests that come directly
+## from the 'self' node. It is expected to attend the store requests that
+## come from REST-store endpoint when those requests don't indicate
+## any store-peer address.
+##
+## Notice that the REST-store requests normally assume that the REST
+## server is acting as a store-client. In this module, we allow that
+## such REST-store node can act as store-server as well by retrieving
+## its own stored messages. The typical use case for that is when
+## using `nwaku-compose`, which spawn a Waku node connected to a local
+## database, and the user is interested in retrieving the messages
+## stored by that local store node.
+##
```

Comment:
Probably need to change this instances of "store" in this comment for "lightpush", and removing the parts that aren't relevant

Reply:
Ah, yes I was using store self query as motivation, but did not check the comment careful.